_posts/2017-02-17-holacracy-and-purpose.md

```diff
@@ -0,0 +1,65 @@
+
+## The concept of purpose
+
+At first, this might feel odd. Why, from a managing director point of view, would you voluntarily choose to give up control over which projects have priority and how they are being executed? To answer this question, we'll first take a closer look at the concept of purpose.
+
+Holacracy [defines purpose as follows](http://www.holacracy.org/constitution#art523):
+
+> (...) the deepest creative potential [an organization] can sustainably express in the world, given all of the constraints acting upon it and everything available to it.
+
+As said earlier, as a company's top-most Lead Link, you are accountable for discovering and expressing the organization's purpose. This means you have a duty to ask this question: what is the deepest creative potential our organization can express in the world? Or put differently: what is the reason our organization exists?
+
+Make no mistake: these are tough questions to answer! Holacracy acknowledges this, and also assumes purpose not to be defined once and never changed, but as evolutionary. Over time, given new information, an organization's purpose will change, and as Lead Link, you may (and should) update the purpose when needed.
+
+## Sizing it up
+
+Time to go back to our thought experiment. While we never mentioned anything about purpose, we did mention three other concepts: strategy, project and action. Each of these can be seen as part of another. Arranged from small to large, this gives us the following sequence:
+
+* The **action** "write draft job advertisement for approval" _is part of_
+* **project** "two new software engineers hired", _which helps execute_
+* **strategy** "increase engineering capacity".
+
+Where would **purpose** fit in this sequence? Right after strategy. Any strategic focus you set is in service of better fulfilling the organization's purpose, just as a project helps execute a strategy and an action helps finish a project. In a way, purpose explains why strategies, projects and actions exist.
+
+## Shared focus
+
+Well, you say, this is all cleverly thought out, but why bother with all this purpose? Why not just stick with delegating projects and actions to people?
+
+While it is beyond the scope of this article to go into all of the reasons behind transitioning to Holacracy and self-management in general, I do like to point out two things in particular: time and brainpower.
+
+It is all fine as a managing director to come up with the projects and actions for your employees to execute, but only up to a point. Once the organization grows larger, you won't have time to do this all as a single person (or you'd have to skip sleeping). This is traditionally the point where management layers enter the arena.
```

Comment:
Dus, management er bij. Waarom is dat verkeerd? En hoe is dat anders dan “lead links er bij”?